src/invert/laplace/impls/cyclic/cyclic_laplace.cxx

```diff
@@ -476,3 +477,74 @@ Field3D LaplaceCyclic::solve(const Field3D& rhs, const Field3D& x0) {
 
   return x;
 }
+
+void LaplaceCyclic :: verify_solution(const Matrix<dcomplex> &a_ver, const Matrix<dcomplex> &b_ver, const Matrix<dcomplex> &c_ver, const Matrix<dcomplex> &r_ver, const Matrix<dcomplex> &x_sol, const int nsys)
+{
+    output.write("Verify solution\n");
+    const int nx = xe - xs + 1;  // Number of X points on this processor,
+                                 // including boundaries but not guard cells
+    const int myrank = localmesh->getXProcIndex();
+    const int nprocs = localmesh->getNXPE();
+    int i;
```

Comment:
Is this needed outside of the loop at the end?

Reply:
Nope, was a dumb copy. Fixed.